utils/host.go

```diff
@@ -1,4 +1,4 @@
-package runtime
+package utils
```

Comment:
Move the file to runtime/utils/ ..similar to diagnostics/utils..

Reply:
That forms a cyclic dependency: this is being used in pkg/http and pkg/runtime depends on pkg/http

Reply:
ok. lgtm then.